testing/describes.js

```diff
@@ -429,7 +429,13 @@ class AmpFixture {
     }
     if (spec.extensions) {
       spec.extensions.forEach(extensionId => {
-        const installer = extensionsBuffer[extensionId];
+        let bufferId = extensionId;
```

Comment:
Not quite. See `extensionId` usage lower in this method. I recommend to do this:

1. Rename `extensionId => ` arg to `bufferId` or `extensionWithVersion` or such.
2. First thing in the method, parse it out into `extensionId` and `version`.
3. Use `${extensionId}:${version}` to lookup installer in the `extensionsBuffer`.
4. For everything else, use `extensionId` by itself.